.github/workflows/pr-code-format.yml

```diff
@@ -71,6 +71,6 @@ jobs:
           python ./code-format-tools/llvm/utils/git/code-format-helper.py \
             --token ${{ secrets.GITHUB_TOKEN }} \
             --issue-number $GITHUB_PR_NUMBER \
-            --start-rev $START_REV \
+            --start-rev $(git merge-base $START_REV $END_REV) \
```

Comment:
Note that this change won't do anything right here as this workflow is setup as a `pull_request_target` for security reasons, which means it uses the workflow definition from the PR target (the `main` branch) rather than the workflow branch. I've opened #75132 which should fix this issue in the CI. Rebasing to the latest `main` should also fix it.

Reply:
OK. I was just trying it out to see if it worked, but I didn't realize it couldn't be tested that way! Thanks.